engine/src/test/java/net/jqwik/api/constraints/WhitespaceProperties.java

```diff
@@ -0,0 +1,24 @@
+package net.jqwik.api.constraints;
+
+import net.jqwik.api.ForAll;
+import net.jqwik.api.Group;
+import net.jqwik.api.Property;
+
+@Group
+class WhitespaceProperties {
+
+	@Property
+	boolean stringWithWhitespace(@ForAll @Whitespace String aString) {
+		return aString.chars().allMatch(Character::isWhitespace);
+	}
+
+	@Property
```

Comment:
The property is a bit too weak, since it does not check that all whitespace characters will be generated at some time. Such properties would cover the problem mentioned above.

Reply:
I already thought of this but didn't have a catchy idea how to achieve it with reasonable effort.  
Any suggestions?

Reply:
`WhitespaceProperties` is fine I think. You should consider to add tests to `DefaultStringArbitraryTests` and `DefaultCharacterArbitraryTests`. The examples there do similar things.

Reply:
Added tests.